Date: 2014-09-12 12:01:59.975770528 +0200
From: Jan Engelhardt <jengelh@inai.de>

__DATE__ and __TIME__ is useless because we may rebuild regularly,
automatically, and when we do, it changes the binary output for
no reason, defeating the "same result" checker.

---
 tools/src/hfst-optimized-lookup.cc |    1 -
 tools/src/hfst-proc/hfst-proc.cc   |    1 -
 2 files changed, 2 deletions(-)

Index: hfst-3.12.2/tools/src/hfst-optimized-lookup.cc
===================================================================
--- hfst-3.12.2.orig/tools/src/hfst-optimized-lookup.cc
+++ hfst-3.12.2/tools/src/hfst-optimized-lookup.cc
@@ -97,7 +97,6 @@ bool print_version(void)
   std::cout <<
     "\n" <<
     PACKAGE_STRING << std::endl <<
-    __DATE__ << " " __TIME__ << std::endl <<
     "copyright (C) 2009 University of Helsinki\n";
   return true;
 }
Index: hfst-3.12.2/tools/src/hfst-proc/hfst-proc.cc
===================================================================
--- hfst-3.12.2.orig/tools/src/hfst-proc/hfst-proc.cc
+++ hfst-3.12.2/tools/src/hfst-proc/hfst-proc.cc
@@ -154,7 +154,6 @@ bool print_version(void)
     " (" << PACKAGE_STRING << ")" <<
 #endif
     std::endl <<
-    __DATE__ << " " __TIME__ << std::endl <<
     "copyright (C) 2009-2011 University of Helsinki\n";
   return true;
 }
